integration/raft/migration_test.go

```diff
@@ -15,6 +15,8 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/hyperledger/fabric/common/channelconfig"
+
```

Comment:
please delete the empty line and run the command 'goimports -l -w'

Reply:
Done.
I'm still working on this PR, the title is updated to be W.I.P.